.github/workflows/main.yml

```
@@ -0,0 +1,29 @@
+name: "Build & Deploy to Netlify"
+on:
+  pull_request:
+  push:
+  schedule:
+  - cron: '0 * * * *'
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checking out the repository
+      uses: actions/checkout@v2
+    - name: Installing Nix
+      uses: cachix/install-nix-action@v7
+    - name: Update content
+      run: UPDATE=1 nix-shell -p gnumake -p curl --command "make update"
+    - name: Building nixos.org
+      run: nix-build && mkdir build && cp -RL ./result/* ./build/
+    - name: Deploy to Netlify
+      uses: nwtgck/actions-netlify@v1.0.5
+      env:
+        NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+        NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+      with:
+        publish-dir: './build'
+        production-branch: 'master'
+        github-token: ${{ secrets.GH_TOKEN }}
```

there is a default token you can use that is able to post comments on the pull request:
```suggestion
        github-token: ${{ secrets.GITHUB_TOKEN }}
```

---

I tried to use that and it didn't work ... it got stuck at fetching the token i think.

---

Was the branch pushed on a forked repo or the nixos-homepage repo?